### DISCUSSION TITLE
GitHub Education not verifying despite being on campus

### DISCUSSION THREAD

**Discussion by rajsinghUSA** [community/community#53459] [GitHub Education]
I am unable to verify despite being on campus. My verification is a letter from my Program Director on school letterhead just like it said. My university ID does not have a date on it (though I did try), which is why I got a letter from the school.

![image](https://user-images.githubusercontent.com/17286177/233741778-d3dd11ef-d5b8-49d5-a938-5397a7f5b924.png)

Please help!

**Comment by samus-aran** [comment]
Hi @rajsinghUSA 👋 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Hope this helps! Let us know how you get along 👍

**Reply by rajsinghUSA** [reply]
Thanks for the reply Ashley!

To provide some clarifications to the points you raised:

- Location sharing was already enabled on my Chrome browser.
- I was connected to my school's wifi in the school's library (eduroam)
- I did not have a VPN running

Please advise, gracias!

**Comment by rajsinghUSA** [comment]
Bump @samus-aran

**Comment by rajsinghUSA** [comment]
Hi, anyone that can help with this, as it's a time sensitive issue.

**Comment by rajsinghUSA** [comment]
@samus-aran would very much appreciate your help please
